parlai/mturk/tasks/acute_eval/run.py

```diff
@@ -0,0 +1,544 @@
+        help='text next to speaker 1 radio button',
+    )
+    argparser.add_argument(
+        '--s2_choice',
+        type=str,
+        default='I would prefer to talk to <Speaker 2>',
+        help='text next to speaker 2 radio button',
+    )
+    argparser.add_argument(
+        '--question',
+        type=str,
+        default='Who would you prefer to talk to for a long conversation?',
+        help='question to present to turker for comparison (e.g. "Which speaker is better?")',
+    )
+    argparser.add_argument(
+        '--correctness_is_flipped',
+        default=False,
+        action='store_true',
+        help='question phrasing flips the better model - e.g. question is "Which speaker is more boring?"',
+    )
+    argparser.add_argument(
+        '--model_comparisons',
+        type=str,
+        help='list of model pairs to compare, comma separated. E.g. ["transformer,human_eval"] ',
+    )
+    argparser.add_argument(
+        '--pairs_per_matchup',
+        type=int,
+        default=160,
+        help='Number of conversation pairs to generate for the comparison',
+    )
+    argparser.add_argument(
+        '--num_onboarding_tasks',
+        type=int,
+        default=5,
+        help='Number of onboarding tasks total to screen workers with',
+    )
+    argparser.add_argument(
+        '--block_on_onboarding_fail',
+        type=bool,
+        default=True,
+        help='whether to block on onboarding failure',
+    )
+    argparser.add_argument(
+        '--onboarding_tasks',
+        type=list,
+        default=None,
+        help='onboarding tasks to screen workers with, list of (conv1id, conv2id, matchup) tuples',
+    )
+    argparser.add_argument(
+        '--onboarding_model_comparison',
+        type=str,
+        default='greedy_model,human_eval',
+        help='models to compare for the onboarding task. E.g. "greedy,human_eval" ',
+    )
+    argparser.add_argument(
+        '--subtasks_per_hit',
+        type=int,
+        default=5,
+        help='number of subtasks/comparisons to do per hit',
+    )
+    argparser.add_argument(
+        '--onboarding_threshold',
+        type=float,
+        default=0.75,
+        help='minimum accuracy on onboarding tasks, as a float 0-1.0',
+    )
+    argparser.add_argument('--seed', type=int, default=42, help='np.random seed')
+    argparser.set_defaults(allowed_conversation=1)
+    if from_argv:
+        return argparser.parse_args()
+    else:
+        return argparser.parse_args(args=[])
+
+
+def list_files(folder):
+    for fn in os.listdir(folder):
+        full_fn = os.path.join(folder, fn)
+        if os.path.isfile(full_fn):
+            yield full_fn
+        elif os.path.isdir(full_fn):
+            for sfn in list_files(full_fn):
+                yield sfn
+
+
+def setup_task_queue(opt):
+    """ Initialize task queue to contain the specified number of instances of
+    each pairing
+    """
+    # hacky fix for the parlai parser hacky fix
```

Comment:
hmm... why aren't the hyphens already replaced?

Reply:
it wasn't at the time I wrote this - I think the hyphen replacing was happening before my args were being added, but then the underscores in my args got converted to hyphens - maybe it's been fixed since.